.eslintignore

```diff
@@ -19,7 +19,7 @@ dist.tools
 export
 examples/storybook
 extensions/**/dist
-release
+/release
```

Comment:
nit: Can we use a trailing slash here if it works? (Then the entry in `clean.js` can be synced with this.)

Reply:
In addition to the preceding slash? (this works) or instead of it? (this doesn't work)

Reply:
I meant trailing only, but realized that wouldn't work because the JS file is not `release.js`, but `release/index.js`. Thanks for humoring me, and carry on!